renderer/components/Settings/SettingsForm.js

```diff
@@ -18,6 +19,11 @@ const SettingsForm = ({
 }) => {
   const handleSubmit = async values => {
     try {
+      // If password feature has been disabled, reset the password to null.
+      if (!get(values, 'password.value')) {
+        values.password.value = null
```

Comment:
Should it be `null` or removed all together with `delete` ?

Reply:
It has to be set back to null. Only keys that exist in the values object will be applied as config overrides when the form is saved.